### PR TITLE
Vote-1194: Arabic translation

### PR DESCRIPTION
**Pull request by mlloydbixal**

### CODE REVIEW

File: data/translations/ar/state-data.json [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    "default_online_deadline": "متاح حتى يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام بحلول يوم الانتخابات أو في يومه"
+  },
+  "va": {
+    "default_ip_deadline": "22 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "22 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 22يومًا من يوم الانتخابات"
+  },
+  "vi": {
+    "default_ip_deadline": "30 يومًا قبل يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:241]: Missing "default_mail_postmarked_deadline"

File: data/translations/ar/state-data.json [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 30 يومًا من يوم الانتخابات"
+  },
+  "sc": {
+    "default_ip_deadline": "30 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "30 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 30 يومًا من يوم الانتخابات"
+  },
+  "sd": {
+    "default_ip_deadline": "15 يومًا قبل يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 15 يومًا من يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:214]: this should be "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+  },
+  "or": {
+    "default_ip_deadline": "21 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "21 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 21 يومًا من يوم الانتخابات"
+  },
+  "pa": {
+    "default_ip_deadline": "15 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "15 يومًا قبل يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 15 يومًا من يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:197]: fix key name "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 10 أيام من يوم الانتخابات"
+  },
+  "nc": {
+    "default_ip_deadline": "التسجيل في نفس اليوم متاح خلال فترة التصويت المبكر",
+    "default_online_deadline": "قبل 25 يومًا من يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 25 يومًا من يوم الانتخابات"
+  },
+  "nd": {},
+  "mp": {
+    "default_ip_deadline": "60 يومًا قبل يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:178]: missing "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+  },
+  "nm": {
+    "default_ip_deadline": "28 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "28 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 28 يومًا من يوم الانتخابات"
+  },
+  "ny": {
+    "default_ip_deadline": "قبل 25 يومًا من يوم الانتخابات",
+    "default_online_deadline": "قبل 25 يومًا من يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 10 أيام من يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:169]: fix key "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+    "default_online_deadline": "5 أيام قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 28 يومًا من يوم الانتخابات"
+  },
+  "nh": {
+    "default_ip_deadline": "متاح حتى يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 13 يومًا من يوم الانتخابات"
+  },
+  "nj": {
+    "default_online_deadline": "21 يومًا قبل يوم الانتخابات",
+    "mail_received_deadline": "يجب الختم بختم البريد قبل 21 يومًا من يوم الانتخابات",
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:158]: fix key "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 18 يومًا من يوم الانتخابات"
+  },
+  "nv": {
+    "default_ip_deadline": "متاح حتى يوم الانتخابات",
+    "default_online_deadline": "5 أيام قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 28 يومًا من يوم الانتخابات"
+  },
+  "nh": {
+    "default_ip_deadline": "متاح حتى يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 13 يومًا من يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:154]: fix key "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+  },
+  "mi": {
+    "default_ip_deadline": "متاح حتى يوم الانتخابات",
+    "default_online_deadline": "15 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 15 يومًا من يوم الانتخابات"
+  },
+  "mn": {
+    "default_ip_deadline": "متاح حتى يوم الانتخابات",
+    "default_online_deadline": "21 يومًا قبل يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 21 يومًا من يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:127]: fix key "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 29 يومًا من يوم الانتخابات"
+  },
+  "la": {
+    "default_ip_deadline": "30 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "20 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 30 يومًا من يوم الانتخابات"
+  },
+  "me": {
+    "default_ip_deadline": "متاح حتى يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 21 يومًا من يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:107]: fix key "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 29 يومًا من يوم الانتخابات"
+  },
+  "ga": {
+    "default_ip_deadline": "29 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "29 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب أن يتم الاستلام قبل 29 يومًا من يوم الانتخابات"
+  },
+  "gu": {
+    "default_online_deadline": "11 يومًا عمل قبل يوم الانتخابات",
+    "mail_received_deadline": "يجب الاستلام قبل 11 يومًا عمل من يوم الانتخابات",
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:62]: fix key "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل7 أيام من يوم الانتخابات"
+  },
+  "de": {
+    "default_ip_deadline": "24 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "24 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل24 يومًا من يوم الانتخابات"
+  },
+  "dc": {
+    "default_ip_deadline": "متاح حتى يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 21 يومًا من يوم الانتخابات",
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:47]: fix key "default_mail_received_deadline"

File: data/translations/ar/state-data.json [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 15 يومًا من يوم الانتخابات"
+  },
+  "ak": {
+    "default_ip_deadline": "30 يومًا قبل يوم الانتخابات",
+    "default_online_deadline": "30 يومًا قبل يوم الانتخابات",
+    "default_mail_postmarked_deadline": "يجب الختم بختم البريد قبل 30 يومًا من يوم الانتخابات"
+  },
+  "as": {
+    "default_ip_deadline": "29 يومًا قبل يوم الانتخابات",
+    "mail_received_deadline": "يجب أن يتم الاستلام قبل 29 يومًا من يوم الانتخابات"
```
Comment (rayestrada) [inline data/translations/ar/state-data.json:14]: fix key "default_mail_received_deadline"

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: data/translations/ar/state-data.json (hunk 1 above)
Comment (rayestrada) [reply data/translations/ar/state-data.json:241]: No longer required.

File: data/translations/ar/error.json [hunk 13]
```diff
@@ -0,0 +1,11 @@
+{
+  "error_main_heading": "عفوا! خطأ 404",
+  "error_intro": "لم يتم العثور على الصفحة",
+  "error_msg": "لا يمكننا العثور على الصفحة التي تبحث عنها. ربما تمت إزالتها أو تغيير اسمها أو أنها غير متاحة مؤقتًا.",
+  "error_msg1": "إذا كتبت عنوان URL للصفحة، فإذا قمت بكتابة عنوان URL للصفحة ، فتحقق من الإملاء.",
```
[the remark this answers is not in the document]
Comment (mlloydbixal) [reply data/translations/ar/error.json:5]: I double checked the translation document and this is what they gave us.. perhaps they are saying "if you typed the page URL check the spelling of the URL" but I ran it through google translate and it sounds duplicated to me. However, I can't make sense of how to remove the duplication. I'll ask Omer to check with the translator.

File: data/translations/ar/homepage.json [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+  "alert__alert_after_election": "للحصول على نتائج الانتخابات الرسمية ، انتقل إلى الموقع  الخاص بالانتخابات في ولايتك.",
+  "header": "سجل للتصويت",
+  "state_selection__heading": "أين تعيش؟",
+  "state_selection__default": "حدد ولايتك أو إقليمك",
+  "state_selection__submit": "تعرف على كيفية التسجيل",
+  "registered__header": "هل أنت مسجل بالفعل؟",
+  "registered__resources_intro": "استكشف %link% على %USA.gov%",
+  "registered__resources_link_text": "مصادر التصويت والانتخابات",
+  "registered__resources_link_url": "https://www.usa.gov/voting",
+  "registered__resources_link_on": " %USA.gov% على",
```
[the remark this answers is not in the document]
Comment (mlloydbixal) [reply data/translations/ar/homepage.json:18]: I'm also removing it for Hindi

**Review by rayestrada — DISMISSED** [review]

**Review by rayestrada — DISMISSED** [review]

**Review by rayestrada — APPROVED** [review]

**Pull request merged**